libr/core/cmd.c

```diff
@@ -153,6 +153,22 @@ static const char *help_msg_star[] = {
 	NULL
 };
 
+static const char *table_help[] = {
+	"Usage:", ",[/] [file]", "# load table data",
+	",", "", "display table",
+	".", "$foo", "aflt > $foo (files starting with '$' are saved in memory)",
+	". ", " [table-query]", "filter and print table. See ,? for more details",
+	"..", " file.csv", "load the csv file into a table",
+	",-", "", "reset table",
+	",/", "?", "query/filter current table",
+	",*", ">$foo", "print table as r2 commands",
+	",j", "", "print table in json format",
+	",h", "xxd foo bar cow", "define header column names and types",
```

Comment:
```suggestion
	",h", " xxd foo bar cow", "define header column names and types",
```
same for other lines